Content.Server/Nutrition/EntitySystems/CreamPieSystem.cs

```diff
@@ -48,7 +48,7 @@ protected override void SplattedCreamPie(EntityUid uid, CreamPieComponent creamP
                 {
                     _puddle.TrySpillAt(uid, solution, out _, false);
                 }
-                if (foodComp.Trash.Count == 0)
+                if (foodComp.Trash.Count != 0)
```

Comment:
Ideally should be upstreamed and then cherry-picked here, but otherwise needs a DeltaV comment.

Reply:
What's the process of cherry-picking like here? I can submit a PR on upstream, and do the cherry-picking part here.

Reply:
Make the PR [upstream](https://github.com/space-wizards/space-station-14/pulls), then cherry pick that the commit into a new branch of your DeltaV fork and make a new PR from it labeled `Early merge "(original PR title)" (#1234)`. Make sure to also link it somewhere in the description.

Reply:
I'm on it :+1: